hangman.js

```diff
@@ -1 +1,21 @@
-console.log("welcome to hangman");
\ No newline at end of file
+console.log("Welcome to Robizzy's HangMan console game.");
+
+var word = prompt("Please enter a word");
+var attempt = [];
+var incorrectAttemptLimit = 7;
+var alphabet = ['a', 'b', 'c', 'd', 'e', 'f', 'g', 'h',
+        'i', 'j', 'k', 'l', 'm', 'n', 'o', 'p', 'q', 'r', 's',
+        't', 'u', 'v', 'w', 'x', 'y', 'z'];
+
+function attempt(letter){
+  if (attempt.includes(letter)){
+    alert("Already attempted this letter");
+    return;
+  }
+}
+
+function countIncorrectAttempts(word, attempts){
+  return attempts.filter(function(count){
+    return !word.match(count);
+  });
+}
```

Comment:
I like the progress so far, would recommend pseudo-coding the next functions/step you would need to complete your game. Would love to see you come back to this, and would to be happy to pair to work on any implementations.